packages/cypress/add-commands/index.d.ts

```diff
@@ -1,12 +1,13 @@
-import type { CommandLoginByOidcOptions } from './dist/commercetools-frontend-cypress-add-commands.cjs';
```

Comment:
Apparently TS is not able to properly import this type, causing the entire declaration file to be unusable.

Using the inline `import()` is the only thing I could find to make it work.